src/components/link/link.pw.tsx

```diff
@@ -0,0 +1,420 @@
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<LinkComponent />, {
+      hooksConfig: {
+        focusRedesignOptOut: true,
+      },
+    });
+    const linkElement = await linkChildren(page);
+    await linkElement.focus();
+    await expect(linkElement).toHaveCSS(
+      "background-color",
+      "rgb(255, 218, 128)"
+    );
+    await expect(linkElement).toHaveCSS("color", "rgba(0, 0, 0, 0.9)");
+    await expect(linkElement).toHaveCSS("border-radius", "2px");
+  });
+
+  test("should render with correct focus state, focusRedesignOptOut false", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkComponent />);
+    const linkElement = await linkChildren(page);
+    await linkElement.focus();
+    await expect(linkElement).toHaveCSS(
+      "background-color",
+      "rgb(255, 218, 128)"
+    );
+    await expect(linkElement).toHaveCSS("color", "rgba(0, 0, 0, 0.9)");
+    await expect(linkElement).toHaveCSS("border-radius", "2px");
+    const linkWrapper = await link(page);
+    await expect(linkWrapper).toHaveCSS(
+      "box-shadow",
+      "rgba(0, 0, 0, 0.9) 0px 4px 0px 0px"
+    );
+  });
+});
+
+test.describe("check events for Link component", () => {
+  test("should call onClick callback when a click event is triggered", async ({
+    mount,
+    page,
+  }) => {
+    let callbackCount = 0;
+    const callback = () => {
+      callbackCount += 1;
+    };
+    await mount(<LinkComponent onClick={callback} />);
+    const linkComponent = await link(page);
+    await linkComponent.click();
+    expect(callbackCount).toBe(1);
+  });
+
+  test("should call onKeyDown callback when a key is pressed", async ({
+    mount,
+    page,
+  }) => {
+    let callbackCount = 0;
+    const callback = () => {
+      callbackCount += 1;
+    };
+    await mount(<LinkComponent onKeyDown={callback} />);
+    const linkComponent = await linkChildren(page);
+    await linkComponent.press("ArrowRight");
+    expect(callbackCount).toBe(1);
+  });
+
+  test("should call onMouseDown callback when a keydown event is triggered", async ({
+    mount,
+    page,
+  }) => {
+    let callbackCount = 0;
+    const callback = () => {
+      callbackCount += 1;
+    };
+    await mount(<LinkComponent onMouseDown={callback} />);
+    const linkElement = await linkChildren(page);
+    await linkElement.dispatchEvent("mousedown");
+    expect(callbackCount).toBe(1);
+  });
+});
+
+test.describe("should check accessibility for Link component", () => {
+  test("should check accessibility for default link component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkComponent />);
+    await checkAccessibility(page);
+  });
+
+  // FE-4647
+  test.skip("should render Link disabled and check accessibility", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkComponent disabled />);
+    await checkAccessibility(page);
+  });
+
+  test.skip("should render Link with dark background and check accessibility", async ({
```

Comment:
similar here and for the other accessibility test, we already checked they should be rendered from the previous functional/prop tests, so I think it would be good just describe the test as "should pass accessibility for ...."